extensions/amp-dynamic-css-classes/0.1/test/integration/test-runtime-classes.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
I had copied that structure from `/test`. Should this be in the extension's top-level test directory instead?